gm4_vigere_shamir/data/gm4_vigere_shamir/functions/update_skull_owner.mcfunction

```diff
@@ -0,0 +1 @@
+ execute if data storage gm4_metallurgy:temp/shamir {name:"vigere"} run data modify storage gm4_metallurgy:temp/shamir skull_owner set value "[Drop to Fix Item] gm4_vigere_shamir:band/v0"
```

Comment:
You don't need this file as your Shamir was created - and will therefore be merged - after PR #593. This is only required to update data on items created before that PR.